sanic/server.py

```diff
@@ -114,6 +114,10 @@ def on_header(self, name, value):
         self.headers.append((name.decode(), value.decode('utf-8')))
 
     def on_headers_complete(self):
+        ra = self.transport.get_extra_info('peername')
```

Comment:
The WSGI frameworks all pull from X-Forwarded-For, which Sanic should do as well.